src/components/Portal/PortalConsumer.js

```diff
@@ -11,7 +11,15 @@ type Props = {
 
 export default class PortalConsumer extends React.Component<Props> {
   componentDidMount() {
-    this._key = this.props.manager.mount(this.props.children);
+    try {
```

Comment:
This might be triggered for errors in rendering the children and not just due to the absence of `Provider`. An explicit check will be better:

```suggestion
    if (!this.props.manager) {
      throw new Error(
        'Looks like you forgot to wrap your root component with `Provider` component from react-native-paper.\n\n' +
          "Please read our getting-started guide and make sure you've followed all the required steps.\n\n" +
          'https://callstack.github.io/react-native-paper/getting-started.html'
      );
    }
    
    this._key = this.props.manager.mount(this.props.children);
  ```

Reply:
Fixed